ironfish-cli/src/commands/wallet/chainport/send.ts

```diff
@@ -0,0 +1,346 @@
+      this.warn(
+        `Transaction '${transaction.hash().toString('hex')}' was not accepted into the mempool`,
+      )
+    }
+
+    const transactionUrl = getExplorer(networkId)?.getTransactionUrl(
+      transaction.hash().toString('hex'),
+    )
+
+    if (transactionUrl) {
+      this.log(`\nIf the transaction is mined, it will appear here: ${transactionUrl}`)
+    }
+
+    if (flags.watch) {
+      this.log('')
+
+      await watchTransaction({
+        client,
+        logger: this.logger,
+        account: from,
+        hash: transaction.hash().toString('hex'),
+      })
+    }
+
+    this.log(
+      `Run ironfish wallet:chainport:status a50d8b0102dbac90f473b8f7192d269a5e5ebc74fa00ac768b065413c8ea82e9 to get the status of your transaction`,
+    )
+  }
+
+  private async constructBridgeTransaction(
+    amount: bigint,
+    assetId: string,
+    selectedNetwork: string,
+    to: string,
+    from: string,
+    client: RpcClient,
+  ) {
+    const url = `https://preprod-api.chainport.io/ironfish/metadata?raw_amount=${amount.toString()}&asset_id=${assetId}&target_network_id=${selectedNetwork}&target_web3_address=${to}`
+
+    this.logger.debug('URL: ' + url)
+
+    const response: {
+      data: BridgeTransactionDetails
+    } = await axios.get(url)
+
+    const outputs: BridgeTransactionDetails = response.data
+
+    this.logger.debug('Outputs: ' + JSON.stringify(outputs))
+
+    const params: CreateTransactionRequest = {
+      account: from,
+
+      outputs: [
+        {
+          publicAddress: outputs.bridge_output.publicAddress,
+          amount: outputs.bridge_output.amount,
+          memoHex: outputs.bridge_output.memoHex,
+          assetId: outputs.bridge_output.assetId,
+        },
+        {
+          publicAddress: outputs.gas_fee_output.publicAddress,
+          amount: outputs.gas_fee_output.amount,
+          memo: outputs.gas_fee_output.memo,
+        },
+      ],
+      expiration: 0,
+    }
+
+    const raw = await selectFee({
+      client,
+      transaction: params,
+      logger: this.logger,
+    })
+    return { outputs, raw }
+  }
+
+  bridgeSummary(
+    outputs: BridgeTransactionDetails,
+    raw: RawTransaction,
+    from: string,
+    to: string,
+    assetId: string,
+    assetData: RpcAsset,
+  ) {
+    const bridgeAmount =
+      BigInt(outputs.bridge_output.amount) - BigInt(outputs.bridge_fee.source_token_fee_amount)
+
+    const bridgeAmountString = CurrencyUtils.render(
+      bridgeAmount,
+      true,
+      assetId,
+      assetData.verification,
+    )
+    const feeString = CurrencyUtils.render(raw.fee, true)
+    const bridgeFeeAmountString = CurrencyUtils.render(
+      BigInt(outputs.bridge_fee.source_token_fee_amount),
+      true,
+      assetId,
+      assetData.verification,
+    )
+    const destinationChainFeeString = CurrencyUtils.render(
+      BigInt(outputs.gas_fee_output.amount),
+      true,
+      assetId,
+      assetData.verification,
+    )
+
+    const summary = `\
```

Comment:
There is a table that is built in to OClif, consider using that as we use it extensively in our other commands.